CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# CODEOWNERS file for /docs/ folder.
+# Forces a review from other writers for anything within /docs/.
+/docs/ @magento/devdocs-admins
```

Comment:
```suggestion
/docs/ @magento/devdocs-admins

```

Reply:
@dobooth please add new line at the end of file